CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.8.1 / 2022-09-23
+
+- [BUGFIX] Add Dashboard ErrorBoundary, fix grid open state #562
```

Comment:
Might want to add https://github.com/perses/perses/pull/547 which introduces a new plugin variable: `StaticListVariable`

Reply:
We'll push that one out in v0.9.0 pretty soon! This was a quick fix so we can manually fix dashboards (errors were making it so we couldn't edit JSON before)